spec/components/search_spec.rb

```diff
@@ -695,6 +695,13 @@ def search
       expect(Search.execute('"3.0, eta is"').posts.length).to eq(0)
     end
 
+    it 'can match exact phrases when they are separated by other than spaces' do
+      topic = Fabricate(:topic, created_at: 3.months.ago)
+      Fabricate(:post, raw: 'this_is_a_test', topic: topic)
+
+      expect(Search.execute('"this is a test"').posts.length).to eq(1)
```

Comment:
We should validate for the exact post here. Someone may add a `let!(:post) { Fabricate(:post) }` outside of this scope and may cause the test to incorrectly pass.